certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -79,7 +80,11 @@ def cleanup():
                 pass
             print('=> Finished tear down of {0} instance.'.format(acme_type))
 
-        shutil.rmtree(workspace)
+        try:
+            shutil.rmtree(workspace)
+        except (OSError, IOError):
+            # Not critical
```

Comment:
I agree this isn't critical, but do you know why this is happening? I've only seen it happen in the oldest tests.

Reply:
Well failure to delete here is due to the OCSP test configuration stored in the assets that contains some files with `0744` permissions, not owned by current user. So current user in Travis fail to delete them. Obviously a `sudo` call would help here, but I wanted to avoid to rely on this, since it is not very portable (for various reasons in a given test env, `sudo` may not be usable/available.

Reply:
Really? Why doesn't the current user own the files?

While this isn't a big deal, I'd ideally like to fix this so the tests properly clean up after themselves.

Reply:
Because it is owned by uid=0 if I remember correctly, so root. I need to check again, but I think it was about sensible files with specific permissions, so it may match some security logic in certbot.

Reply:
Ok, it seems that we can forget about this, the local copy done by `shutil.copytree` is entirely cleanable by the current user from the tests I did locally and on Travis. So I removed the try/except.